index.html

```diff
@@ -147,40 +169,58 @@ <h2>Uploading New Materials</h2>
+					<img src="./Resources/CLARK-Draft-Edit.png" width="1000px">
+					<p style="padding-top: 20px;">Note: you can only edit a learning object that is in an unreleased state. Will will not have the option to edit learning objects that are waiting for review, under review, in proofing, or denied.</p>
 				</div>
 			</div>
 			<br><br>
 
-			<a name="Children"><div class="section">
-				<h2>Adding Children Learning Objects</h2>
+			<a name="Revisions"><div class="section">
+				<h2>Revising Released Learning Objects</h2>
 				<hr>
-				<p>When you upload a large learning object that uses many smaller objects in it, it is useful to add these objects as <i>children</i> to the larger one.  This is to help organize your larger learning objects and allow downloaders to easily find all the parts of your curriculum.</p>
-				<p>To start adding child objects go to your dashboard, click on the meatball menu of the parent object, and click the option that says <i>Edit Children.</i>  From here you have the option to choose which learning objects are the children at that object.  Note that the option to add children is available for all lengths of objects excluding nanomodules.  It is important to only add the direct children of the specific learning object you are working on.</p>
+				<p>Navigate to your dashboard and select the <i>Released</i> tab. Click the 'meatball menu' (...) of the object you want to revise and select the <i>info</i> option.</p> 
 				<div class="picture">
-					<img src="./Resources/CLARK-Child-Objects.png" width="1000px">
-					<p>The image above is a good example of what to follow when creating the hierarchy for your learning objects.  You can edit an object&#39;s children at anytime, simply click on the meatball menu and reselect the children learning objects.</p>
+					<img src="./Resources/CLARK-editing-partone.png" width="1000px">
 				</div>
+				<p style="padding-top: 40px;">A side panal will appear as show in the image below. Under the Revision section, click the <i>Create Revision</i> button. Once the revision is created, click the meatball menu in the Revision section. Click the option to edit the learning object. You will be redirected to the learning object creation page.</p>
+				<div class="picture">
+					<img src="./Resources/CLARK-Create-Revision.png" width="1000px">
+				</div>
+				<p style="padding-top: 40px;">You can edit or change as much as you desire. The learning object builder automatically saves your work, similar to a Google Document. Once you are finished with making revisions, click the <i>Submit for Review</i> button. You can submit your revisions for review by either selecting the <i>Submit for Review</i> from the Revisions meatball menu while on the info panal or while in the learning object builder itself. You will need to provide a description of what changes were made in the dialog prompt. Your learning object will need to be approved before it can be released with the revisions you made.</p> 
```

Comment:
Didn't you already state this sentence above?

Reply:
Different section; this is revisions while the other was edits. Edits and revisions are two different processes.